Model/Sync/Reset/Main.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace Yotpo\Core\Model\Sync\Reset;
+
+use Magento\Framework\App\ResourceConnection;
+use Yotpo\Core\Model\Config as CoreConfig;
+use Yotpo\Core\Model\AbstractJobs;
+
+class Main
+{
+    const DELETE_LIMIT = 10000;
+
+    /**
+     * @var int
+     */
+    protected $storeId = 0;
+
+    /**
+     * @var CoreConfig
+     */
+    protected $coreConfig;
+
+    /**
+     * @var ResourceConnection
+     */
+    protected $resourceConnection;
+
+    /**
+     * @var AbstractJobs
+     */
+    protected $abstractJobs;
+
+    /**
+     * @var array <string>
+     */
+    protected $cronJobCodes = [''];
+
+    /**
+     * @param ResourceConnection $resourceConnection
+     * @param CoreConfig $coreConfig
+     * @param AbstractJobs $abstractJobs
+     */
+    public function __construct(
+        ResourceConnection $resourceConnection,
+        CoreConfig $coreConfig,
+        AbstractJobs $abstractJobs
+    ) {
+        $this->resourceConnection = $resourceConnection;
+        $this->coreConfig = $coreConfig;
+        $this->abstractJobs = $abstractJobs;
+    }
+
+    /**
+     * @param int $storeId
+     * @return void
+     */
+    public function resetSync($storeId)
+    {
+        $this->removeCronJobs();
```

Comment:
Please rename to a more revealing name.
```suggestion
        $this->deleteCronSchedules();
```

Reply:
This is done. Will update the PR soon